diff --git a/pkg/oc/cli/admin/router/router.go b/pkg/oc/cli/admin/router/router.go
index 98f74ff..ef7b847 100644
--- a/pkg/oc/cli/admin/router/router.go
+++ b/pkg/oc/cli/admin/router/router.go
@@ -908,7 +908,14 @@ func RunCmdRouter(f kcmdutil.Factory, cmd *cobra.Command, out, errout io.Writer,
 				Requests: kapi.ResourceList{
 					kapi.ResourceCPU:    resource.MustParse("100m"),
 					kapi.ResourceMemory: resource.MustParse("256Mi"),
+					"qat.intel.com/cy1_dc0": resource.MustParse("1"),
 				},
+				Limits: kapi.ResourceList{
+					"qat.intel.com/cy1_dc0": resource.MustParse("1"),
+				},
+			},
+			SecurityContext: &kapi.SecurityContext {
+				Capabilities:   &kapi.Capabilities{Add: []kapi.Capability{"IPC_LOCK"}},
 			},
 		},
 	}
